Sources/TABResourceLoader/Protocols/NetworkResourceType.swift

```diff
@@ -8,6 +8,8 @@
 
 import Foundation
 
+
+/// An enum representing the HTTP Method
 public enum HTTPMethod: String {
   case GET
```

Comment:
I think this should be done in another PR since it'll be an unrelated breaking change